README.md

```diff
@@ -14,7 +14,41 @@ Accessible modal dialog component for React.JS
   <p>Etc.</p>
 </Modal>
 ```
+## Styles
+Styles are passed as an object with 2 keys, 'overlay' and 'content' like so
+```js
+{
+  overlay : {
+    position          : 'fixed',
+    top               : 0,
+    left              : 0,
+    right             : 0,
+    bottom            : 0,
+    'backgroundColor' : 'rgba(255, 255, 255, 0.75)'
```

Comment:
Does this key have to be quoted? Wondering if it's some quirk that the key has to be quoted because of the casing. If not, you should get rid of the quoted key here (and in the content section below).